### DISCUSSION TITLE
百度网盘秒传功能无效

### DISCUSSION THREAD

**Discussion by zhaoshanzeng** [AlistGo/alist#5978] [Q&A]
## alist信息：
![image](https://github.com/alist-org/alist/assets/40257744/90454dae-219a-47bc-8664-c0c7cf92b1ad)

## 部署环境：
群晖套件版

## 问题
把群晖本地和百度网盘添加到alist，在alist web上从本地复制文件到百度网盘，无法秒传。

## 测试信息：
在百度网盘里面有夺宝奇兵电影文件（确保百度网盘是有这个文件的）
![image](https://github.com/alist-org/alist/assets/40257744/49b79e45-9db1-4972-8484-f9c1cacd2484)
现在本地的夺宝奇兵上传到另一个文件夹，没有秒传，走的是网络上传，速度很慢。
![image](https://github.com/alist-org/alist/assets/40257744/57cafa79-5a95-4a84-9e81-8f35ec076c85)

文件重命名过，确定是同一个文件，大小相同，而且即使我没有上传到我的网盘里，这个电影大概率在百度网盘也是有的。
![image](https://github.com/alist-org/alist/assets/40257744/2f46ff65-e684-421d-b73c-21953b51d2c7)

再次测试，同一个文件两次上传到不同目录，第二次上传不是秒传
![image](https://github.com/alist-org/alist/assets/40257744/aa7b2cf1-f74c-4484-b37f-57ad9cc9b472)

**Comment by xhofe** [comment]
https://pan.baidu.com/union/doc/3ksg0s9r7

百度网盘已不支持秒传。
